### PR TITLE
redirect to relative path

### PR DESCRIPTION
**Pull request by dlyongemallo**

### CODE REVIEW

**Comment by keghani** [comment]
I'll wait until 7.2.3 is complete before merging this.

File: java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java [hunk 1]
```diff
@@ -106,7 +106,8 @@ public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOEx
     if (fileContents.length() == 0) {
       // Redirect to a URL with the given input encoded in the query parameters.
       Locale geocodingLocale = new Locale(languageCode, regionCode);
-      resp.sendRedirect(getPermaLinkURL(phoneNumber, defaultCountry, geocodingLocale));
+      resp.sendRedirect(getPermaLinkURL(phoneNumber, defaultCountry, geocodingLocale,
+            false /* absoluteURL */));
```
Comment (keghani) [inline java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java]: indentation

**Comment by keghani** [comment]
Tested a clone of dlyongemallo:master, this works.
Please fix indentation then I'll merge.

File: java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java (hunk 1 above)
Comment (dlyongemallo) [inline java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java]: done

**Pull request merged**